ruby/puppet

```diff
@@ -1 +1 @@
-Subproject commit 40e76e807f314e9435546f57013d66dd67e0d786
+Subproject commit b792d8437f925a640f74bc1642fb6a129b276e72
```

Comment:
Was this change intentional?  Seems like we still want to have the stable branch pinned to Puppet 3.7.4 for now?